adcc/__init__.py

```diff
@@ -79,6 +79,12 @@ def adc0(*args, **kwargs):
     return run_adc(*args, **kwargs, method="adc0")
 
 
+@with_runadc_doc
+def cis(*args, **kwargs):
+    state = run_adc(*args, **kwargs, method="adc1")
+    return ExcitedStates(state, property_method="adc0")
```

Comment:
Ok. That's not fully what I had in mind, actually. I thought we would:
```
    state = run_adc(*args, **kwargs, method="adc1")
    state.property_method = "adc0"
    return state
```
Or do you see any advantages of your current solution? I haven't read the full PR. Is [this](https://github.com/adc-connect/adcc/pull/56/commits/187ef276b902f8dcf15b6202714f4a9f73107e15#diff-e59f3397b8a901324480f99b56e5f5cf) the only change you needed in ExcitedStates? Because if it is I think I am ok with your current solution as well. Will do a full sweep of the PR later today btw.

Reply:
Yeah, it's the only change in ExcitedStates.
For the other solution you posted above, we would need to invalidate all computed densities and cached properties,  which I'd rather do in a separate PR.